.github/workflows/agw-workflow.yml

```diff
@@ -181,33 +182,33 @@ jobs:
           BAZEL_CACHE_CUTOFF_MB: 6000
         run: |
           ./.github/workflows/check-bazel-cache-dir-size.sh "$BAZEL_CACHE_DIR" "$BAZEL_CACHE_CUTOFF_MB"
-      - name: Setup Devcontainer Image
+      - name: Setup Bazel Base Image
         uses: addnab/docker-run-action@v2
         with:
-          image: ${{ env.DEVCONTAINER_IMAGE }}
+          image: ${{ env.BAZEL_BASE_IMAGE }}
           # Run a simple echo command to pull down the image. This makes it a bit more clear how much time is spent on building Magma and not pulling down the image.
           run: |
             echo "Pulled the devontainer image!"
       - name: Create a directory to store XML test results
         uses: addnab/docker-run-action@v2
         with:
           shell: bash
-          image: ${{ env.DEVCONTAINER_IMAGE }}
+          image: ${{ env.BAZEL_BASE_IMAGE }}
           # TODO: Remove work-around mount of Github workspace to /magma (https://github.com/addnab/docker-run-action/issues/11)
-          options: -v ${{ github.workspace }}:/workspaces/magma/ -v ${{ github.workspace }}/lte/gateway/configs:/etc/magma
```

Comment:
Why do we need to remove the `workspaces` from the path? Is it because the `bazel-base` image does not use this concept?

Reply:
The bazel-base `MAGMA_ROOT` is at `/magma`, for devcontainer it's `/workspace/magma`.

Reply:
It might be easier to change the bazel-base container to also use `/workspace/magma` or update the devcontainer to use just `/magma` to avoid switches.
Updating the bazel-base image seems currently easier to me though.

Reply:
Yes, it should be unified. I would move this out of scope here and suggest to do this in the effort to make the devcontainer build on top of the bazel base image. Sounds good?